src/main/ac-context.ts

```diff
@@ -24,20 +22,37 @@ export class AutomationCloudContext {
 
     requireOrganisationId(): string {
         if (!this.organisationId) {
-            throw new Exception({
-                name: 'AuthenticationError',
-                message: 'OrganisationId is required',
-                status: 401,
-            });
+            throw AuthenticationError('OrganisationId is required');
```

Comment:
So. We throw `AuthenticationError` everywhere here. There's a difference though:

- 401 indicates that "authentication is required" which usually prompts to obtain credentials (browsers display popup for basic auth for example); this happens when auth credentials are not provided
- when incorrect credentials are provided, we can choose to throw something else, e.g. `400` if jwt is malformed and 403 if *authentication* is successful but `organisationId` does not exist (because this is no longer the *authentication* part of the *authorisation* process).